common/containers/Tabs/ScheduleTransaction/components/ScheduleFields.scss

```diff
@@ -0,0 +1,64 @@
+@import 'common/sass/variables';
+@import 'common/sass/mixins';
+
+.ScheduleFields {
+  padding: 0 20px 20px 20px;
+  border: 1px solid #e5ecf3;
+
+  &-title {
+    margin-top: -10px;
+    background: white;
+    padding: 0 15px;
+    display: block;
+    width: 250px;
+  }
+
+  &-logo {
+    float: right;
+    max-width: 160px;
+    width: 100%;
+    height: 32px;
+    background-image: url(~assets/images/chronologic-logo.svg);
+    background-repeat: no-repeat;
+    display: block;
+    background-size: cover;
+  }
+
+  &-description {
+    font-weight: 300;
+    font-size: 16px;
+    padding: 20px 0 30px;
+    text-align: justify;
+  }
+
+  &-field-title {
+    position: relative;
+    white-space: nowrap;
```

Comment:
I think this may have caused some kind of issue with the tooltips:

![screen shot 2018-04-12 at 1 42 25 pm](https://user-images.githubusercontent.com/649992/38694490-c6dc3cf0-3e57-11e8-8741-c946ab09760f.png)

Reply:
Fixed in: https://github.com/MyCryptoHQ/MyCrypto/pull/1343/commits/5624221d8ebeef5ab329a629b129e1b8ffab5ffe